cmd/state-svc/main.go

```diff
@@ -203,10 +202,7 @@ func runForeground(cfg *config.Instance, an *anaSvc.Client) error {
 }
 
 func runStart(out output.Outputer) error {
-	exec := appinfo.SvcApp().Exec()
-	ns := svcctl.NewIPCNamespaceFromGlobals()
-	ipcClient := ipc.NewClient(ns)
-	if _, err := svcctl.EnsureAndLocateHTTP(ipcClient, exec); err != nil {
+	if _, err := svcctl.DefaultEnsureStartedAndLocateHTTP(); err != nil {
```

Comment:
I don't understand what `Default` is meant to convey here? I'd prefer we suffix the non-default rather than prefix the default.